cloudygames/models.py

```diff
@@ -8,15 +18,68 @@ class Game(models.Model):
     address = models.CharField(max_length=45)
     users = models.ManyToManyField(User)
 
+class GameSession(models.Model):
+    player = models.ForeignKey(User)
+    game = models.ForeignKey(Game)
+    controller = models.IntegerField()
+
+    def joinGame(self, _game):
```

Comment:
use PEP8 naming convention, use **under_score** for separation. `join_game`, **camelCase** is Java convention, don't mixed up